modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -198,47 +121,6 @@ def try_recompute_cache(indices, previous_cache):
         new_obj._width_cache = try_recompute_cache(col_labels, self._width_cache)
         return new_obj
 
-    @classmethod
-    def put(cls, obj):
-        """
-        Put an object into a store and wrap it with partition object.
-
-        Parameters
-        ----------
-        obj : object
-            An object to be put.
-
-        Returns
-        -------
-        PandasDataframePartition
-            New `PandasDataframePartition` object.
-        """
-        pass
-
-    @classmethod
-    def preprocess_func(cls, func):
-        """
-        Preprocess a function before an `apply` call.
-
-        Parameters
-        ----------
-        func : callable
-            Function to preprocess.
-
-        Returns
-        -------
-        callable
-            An object that can be accepted by `apply`.
-
-        Notes
-        -----
-        This is a classmethod because the definition of how to preprocess
-        should be class-wide. Also, we may want to use this before we
-        deploy a preprocessed function to multiple `PandasDataframePartition`
-        objects.
-        """
-        pass
-
     @classmethod
     def _length_extraction_fn(cls):
```

Comment:
Should we really have this function?

Reply:
Looks good. We could probably preprocess these functions at the class level. However, the perf should be checked.

Reply:
Split the changes into https://github.com/modin-project/modin/pull/4271